src/PresentationalComponents/RulesTable/RulesTable.js

```diff
@@ -173,60 +153,12 @@ const RulesTable = ({ isTabActive }) => {
     setFilters({ ...filters, offset: newOffset });
   };
 
-  const toggleRulesDisabled = (rule_status) => {
```

Comment:
I would vote for leaving functions, like this, that are specific to a component inside. This function deals with updating a state only. Functions that can be reused in other components are great to be refactored

Reply:
This makes sense, I've moved it back and just passed it into the filter config for the onChange function